runtime/Makefile.rules

```diff
@@ -4,12 +4,12 @@ MAKEFILE_RULES_DIRNAME := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 
 # NOTE: we are using  absolute paths from the Docker container
 SNITCH_LLVM_PATH = /opt/snitch-llvm
-SNITCH_RTL_PATH = /opt/snitch_cluster
+SNITCH_SW_PATH = /opt/snax-mac
 CC            = $(SNITCH_LLVM_PATH)/bin/riscv32-unknown-elf-clang
 LD            = $(SNITCH_LLVM_PATH)/bin/riscv32-unknown-elf-clang
 AR            = $(SNITCH_LLVM_PATH)/bin/llvm-ar
 RANLIB        = $(SNITCH_LLVM_PATH)/bin/llvm-ranlib
-VLTSIM        = /opt/snitch-rtl/bin/snitch_cluster.vlt
+VLTSIM        = /opt/snax-mac-rtl/bin/snitch_cluster.vlt
```

Comment:
If the correct `VLTSIM` is now selected by the kernel, does it make sense to define this parameter seperately in every kernel `Makefile`, instead of the common one? I don't think it is used in the common `Makefile.rules` anyway.

Reply:
@jorendumoulin How do you feel about #38 ?

Reply:
#38 is great!